src/styles/calendary.scss

```
@@ -0,0 +1,50 @@
+.calendary {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-size;
+  flex-direction: row;
+  gap: $gap;
+  padding: $padding;
+
+  @each $day, $index in $start-days {
+    &--start-day-#{$day} &__day:first-child {
+      margin-left: ($day-size + $gap ) * $index;
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index} :nth-child(n + #{$index + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    width: $day-size;
+    height: $day-size;
+    justify-content: center;
+    align-items: center;
+
+    font-size: 30px;
+    font-family: Arial, sans-serif;
+
+    background-color: $color_background;
+    border: $border-width;
+
+    transition: transform 0.5s;
```

```suggestion
    transition: transform 0.5s, background-color 0.5s;
```
You can apply transition for other properties too, so background-color changes smoothly too